examples/workflow-glsp/src/workflow-views.tsx

```diff
@@ -65,6 +65,21 @@ export class ForkOrJoinNodeView extends RectangularNodeView {
     }
 }
 
+@injectable()
```

Comment:
The view definition should be part of `@eclipse-glsp/client` to enable generic reuse.